tests/cli/Cest2Test.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Class CestTest
+ */
+class Cest2Test extends \Codeception\Test\Unit
+{
+    public static function tearDownAfterClass(): void
+    {
+        static $counter = 0;
+        $counter++;
+        if ($counter > 1) {
+            debug_print_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
+            die('2 times cli');
+        }
+        // die('1 time cli');
+        parent::tearDownAfterClass(); // TODO: Change the autogenerated stub
+    }
+
+    /**
+     * @group core
+     */
+    public function testCestNamings()
+    {
+        $klass = new stdClass();
+        $cest = new \Codeception\Test\Cest($klass, 'user', 'tests/cli/BootstrapCest.php');
+
+        $path = 'tests' . DIRECTORY_SEPARATOR . 'cli' . DIRECTORY_SEPARATOR;
```

Comment:
In other tests I have seen that this is done to avoid repeating the constant:
```php
$ds = DIRECTORY_SEPARATOR;
$path = "test{$ds}cli{$ds}";
```

Reply:
I think repeating of constants is ok for 2 times only...

Reply:
this file was committed by accident, I deleted it already